site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -89,7 +88,7 @@ class OAuth2Controller(
           val htmlUrl = (response.json \ "html_url").as[String]
           val email = (response.json \ "email").asOpt[String]
 
-          UserDoobieStore.getOrCreate(
+          UserRepository.instance.getOrCreate(
```

Comment:
We should summon the instance via implícits so we actually test the real implementation if it ever changes. As it stands this is just getting the actual implementation without DI.